doc/release-notes-30352.md

```diff
@@ -0,0 +1,10 @@
+P2P and network changes
+-----------------------
+
+- Pay To Anchor(P2A) are a new standard witness output type for spending,
+  and new a recognised output template. This allows for a key-less anchor
```

Comment:
In 8d956ccb86d70e2059a5e2ccd8e63b7f7c8e3842 "Add release note for P2A output feature"

nit: typos

```suggestion
  and a newly recognised output template. This allows for key-less anchor
```

Reply:
done